### PR TITLE
Added red border on select input fields and removed MUI

### LINKED ISSUE #5055: Add Red border color for all components on required error
**Describe the bug**
Whenever there is error in form submission, the form fields are highlighted in red but few fields are still black.

**To Reproduce**

1. Go to https://care.coronasafe.in/facility/6fa5e305-7eea-4b48-b973-e0679e30f63e/patient
2. Leave the form empty and click `Add Patient`
3. See error

**Expected behavior**
All fields border should be changed to red if the error exists.

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #5055 
Added Red border for errors and removed MUI on forms

- [x] Add Patient under Facility form
http://localhost:4000/facility/56de8619-ac54-4517-b38b-93b4f83249c3/patient
Before:
image

After:
image

- [x] Add Bed Capacity form
http://localhost:4000/facility/56de8619-ac54-4517-b38b-93b4f83249c3/
image

- [x] Create Assets form
http://localhost:4000/facility/56de8619-ac54-4517-b38b-93b4f83249c3/assets/new
image

- [x] Add User form
http://localhost:4000/users/add
image

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by Pranshu1902** [comment]
waiting for #5054

**Review by rithviknishad — COMMENTED** [review]
Prefer using `SelectFormField` over `SelectMenuV2`

File: src/Components/Form/FormFields/SelectFormField.tsx [hunk 1]
```diff
@@ -34,6 +35,7 @@ export const SelectFormField = <T, V>(props: SelectFormFieldProps<T, V>) => {
         optionDescription={props.optionDescription}
         optionIcon={props.optionIcon}
         optionValue={props.optionValue}
+        requiredError={props.requiredError}
```
Comment (rithviknishad) [inline src/Components/Form/FormFields/SelectFormField.tsx]:
```suggestion
        requiredError={props.required && field.error}
```

File: src/Components/Form/FormFields/SelectFormField.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ type SelectFormFieldProps<T, V = T> = FormFieldBaseProps<V> & {
   optionDescription?: OptionCallback<T, React.ReactNode>;
   optionIcon?: OptionCallback<T, React.ReactNode>;
   optionValue?: OptionCallback<T, V>;
+  requiredError?: boolean;
```
Comment (rithviknishad) [inline src/Components/Form/FormFields/SelectFormField.tsx]:
```suggestion
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
Why would you want to pass a requiredError separately when you can derive whether it's a required field and it has an error or not?

**Comment by Pranshu1902** [comment]
> Why would you want to pass a requiredError separately when you can derive whether it's a required field and it has an error or not?

@rithviknishad removed `requiredError`

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@Pranshu1902  @rithviknishad 
image

- [x] the ward is not a mandatory field
- [x] the date is currently shown in yyyy-mm-dd , and not in dd-mm-yyyy format which is mentioned in the error message

**Comment by Pranshu1902** [comment]
@nihal467 Bug resolved
image

**Comment by nihal467** [comment]
LGTM

File: src/Components/Users/UserAdd.tsx [hunk 3]
```diff
@@ -487,7 +487,7 @@ export const UserAdd = (props: UserProps) => {
           return;
         case "email":
           if (
-            state.form[field].length &&
+            state.form[field].length == 0 ||
```
Comment (khavinshankar) [inline src/Components/Users/UserAdd.tsx]:
```suggestion
            state.form[field].length === 0 ||
```

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**